Source/Svg.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.2;net35</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.2;net35;net452;net472</TargetFrameworks>

```

Comment:
Why you needed to add these? The 3.5 version should be compatible with later versions.

Reply:
ObservableCollections is not in the same assembly depending of the targetted framework. With the 3.5 it is in WindowsBase.dll. In 4.x it is in System.ObjectModel.
Without adding these frameworks, an application targetting 4.7.2 for example need to explicitly references the WindowsBase.dll.

Reply:
Ok, I see - makes sense.

Reply:
And to complete: as WindowsBase.dll is not in the Framework 4.x, to use WindowsBase.dll we have to provide a full path to the assembly! So, adding new target framework here seems a better approach to me.

Reply:
Yes, I agree. I also just read up about `AutoGenerateBindingRedirects`, and I think it makes sense to let this switched on, so I will merge this as is.